packages/wp-dashboard/src/components/editorSettings/utils/validateShopifyToken.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export default function isValidShopifyToken(value = '') {
+  const pattern = /^([a-zA-Z0-9_-]){20,50}$/;
```

Comment:
Is there a pattern or min max length we should look for here?

I did some quick searching and haven't found anything yet.

Reply:
Not aware of one, but perhaps here it makes more sense to try and fetch actual products from the API to see if the credentials work?
Might have UX implications though.
Let‘s discuss next week.

Reply:
> fetch actual products from the API to see if the credentials work

I like that idea or something like that .... 

If we accept a key as valid it does give the impression the values are okay.

Another option could be a "test credentials" button or something.

Reply:
Yeah a test button could work too.

Sounds like better suited for a follow-up ticket though.

And then probably best to just remove this token validation and just keep validation for the hostname.

Reply:
Removed and added a todo ---

https://github.com/GoogleForCreators/web-stories-wp/pull/11218/files#diff-17f10d1d54d5997bf00fc4cadc87835d6fcc5b5326d706336d520e3f6637377eR131